app/views/users/show.html.erb

```diff
@@ -9,7 +9,7 @@
     <div class="d-flex justify-content-end" style="width: 350px">
       <p class="card-text">
         <small class="text-muted"
-          >Number of posts: <%= @user.posts_counter %></small
+          >Number of posts: <%= @user.postsCounter %></small
         >
       </p>
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/206526448-7695d6f4-44fc-4e32-8d50-a0dc0861ac6e.png)

**[Optional]**
- [ ] Your link to license file is not working. :fearful: Kindly fix this link to make your readme file descriptive and license file easily accessible for all users :+1: